pkg/ext/extension_apiserver.go

```diff
@@ -53,7 +53,7 @@ const (
 	// The main kube-apiserver will connect to that port (through a tunnel).
 	Port              = 6666
 	APIServiceName    = "v1.ext.cattle.io"
-	TargetServiceName = "imperative-api-extension"
+	TargetServiceName = "api-extension"
```

Comment:
We should delete the older `Service` (`imperative-api-extension`). While we could do it in the init code below, I think it would be worth checking with @JonCrowther if that's a good use case for the migration frameworks that he's working on.

If not, then we can just delete the service and ignore if it's not found.